EIPS/eip-1654.md

```diff
@@ -0,0 +1,123 @@
+---
+eip: 1654
+title: Dapp-wallet authentication process with contract wallets support
```

Comment:
Hm, perhaps would it be possible to make this title more cohesive with the "simple summary"?